src/odemis/acq/fastem.py

```diff
@@ -783,31 +785,33 @@ def correct_beam_shift(self):
         good beam positions we calculate in what direction and how much to shift beams, such that they are always
         centered on the mppc detector.
         """
+        pixel_size = self._ccd.pixelSize.value
+        magnification = self._lens.magnification.value
+        refractive_index = self._lens.refractiveIndex.value
+        numerical_aperture = self._lens.numericalAperture.value
+        wavelength = 550e-9
+        sigma = (magnification / pixel_size) * synthetic.psf_sigma_wffm(
+            refractive_index, numerical_aperture, wavelength
+        )
```

Comment:
@effting is this the correct way of calculating sigma?

Reply:
It should be better to use `ccd.pointSpreadFunctionSize` from `src/odemis/model/_components.py`

Reply:
Could also be `self._ccd.spotSize`. Need to check which one to use

Reply:
Indeed, should be `self._ccd.pointSpreadFunctionsize.value`